react-example/src/views/EmbeddedMsgs.tsx

```diff
@@ -141,98 +124,36 @@ export const EmbeddedMsgs: FC<Props> = () => {
       >
         {messages.length > 0 ? (
           messages.map((message: any, index: number) => {
-            const data = message?.elements;
-            const primaryButton = data?.buttons?.[0];
-            const secondaryButton = data?.buttons?.[1];
-            const defaultAction = data?.defaultAction;
+            const data = message;
 
             switch (selectedButtonIndex) {
               case 0:
                 return (
                   <Card
                     key={index.toString()}
-                    cardStyle={{ margin: 0 }}
-                    title={data?.title}
-                    text={data?.body}
-                    imgSrc={data?.mediaUrl}
-                    primaryBtnLabel={primaryButton?.title}
-                    secondaryBtnLabel={secondaryButton?.title}
-                    onClickPrimaryBtn={() =>
-                      handleOpenUrl(
-                        primaryButton?.action?.type,
-                        primaryButton?.action?.data
-                      )
-                    }
-                    onClickSecondaryBtn={() =>
-                      handleOpenUrl(
-                        secondaryButton?.action?.type,
-                        secondaryButton?.action?.data
-                      )
-                    }
-                    onClickView={() =>
-                      handleOpenUrl(defaultAction?.type, defaultAction?.data)
-                    }
+                    parentStyle={{ margin: 0 }}
+                    messageData={data}
```

Comment:
Could we clarify this? Maybe call the Card's attribute `message` (instead of `messageData`) and then pass in a `message`, rather than `data`? The idea here is that we're passing in the message to be rendered... I think talking about "data" removes a bit of clarity

Reply:
Done